packages/design-system/src/components/Button/Button.jsx

```diff
@@ -159,7 +164,7 @@ Button.propTypes = {
   /**
    * A string corresponding to the button-component variation classes
    */
-  variation: PropTypes.oneOf(['primary', 'danger', 'success', 'transparent']),
+  variation: PropTypes.oneOf(['primary', 'success', 'transparent']),
```

Comment:
Removing `danger` variation will break existing code that are using it. We can remove it when it's fully deprecated.
We want to keep `danger` variation and just throw console.warn deprecated for now (like what you have further up).

Reply:
Maybe add comment line above the `variation` propstype?

/* The `danger` variation is deprecated and will be removed in a future release